shared/pages/Swap/SwapList/SwapList.js

```diff
@@ -51,87 +79,71 @@ export default class SwapList extends Component {
         }
 
         {
-          this.props.data.step >= 2 || (this.props.data.step > 2 && this.props.data.step <= 8) ? (
-            <div style={{ paddingTop: isMobile ? '50px' : '' }} styleName={this.props.data.step >= 2 && this.props.data.step < 5  ? 'stepItem active' : 'stepItem active checked'}>
-              <span styleName="stepNumber">{this.props.data.step >= 2 && this.props.data.step < 5 ? '2' : <i className="fas fa-check" />}</span>
+          this.state.flow.step >= 2 || (this.state.flow.step > 2 && this.state.flow.step <= 8) ? (
+            <div style={{ paddingTop: isMobile ? '50px' : '' }} styleName={this.state.flow.step >= 2 && this.state.flow.step < 5  ? 'stepItem active' : 'stepItem active checked'}>
+              <span styleName="stepNumber">{this.state.flow.step >= 2 && this.state.flow.step < 5 ? '2' : <i className="fas fa-check" />}</span>
               <p styleName="stepText">
-                <FormattedMessage
-                  id="BtcToEthToken58"
-                  defaultMessage="Bitcoin deposition" />
+                <FormattedMessage id="swapList97" defaultMessage="{Currency} deposition" values={{ Currency: sellCurrency === 'BTC' ? sellCurrency : buyCurrency }} />
```

Comment:
```suggestion
                <FormattedMessage id="swapList97" defaultMessage="{Currency} deposit" values={{ Currency: sellCurrency === 'BTC' ? sellCurrency : buyCurrency }} />
```

deposition это другое слово, очень плохо подходит сюда